packages/web-components/src/tabs/tabs/tabs.options.ts

```diff
@@ -0,0 +1,31 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+export const TabListAppearance = {
+  subtle: 'subtle',
+  transparent: 'transparent',
+} as const;
+
+export type TabListAppearance = ValuesOf<typeof TabListAppearance>;
+
+export const TabListSize = {
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+} as const;
+
+export type TabListSize = ValuesOf<typeof TabListSize>;
+
+export const TabListOrientation = {
+  vertical: 'vertical',
+  horizontal: 'horizontal',
+} as const;
+
+export type TabListOrientation = ValuesOf<typeof TabListOrientation>;
+
+export interface TabData {
+  id: string;
+  x: number;
+  y: number;
+  height: number;
+  width: number;
+}
```

Comment:
From what I saw this was internals, do we need to export an interface to consumers for this?